### ISSUE TITLE
How to get logits of validation/test?

### ISSUE THREAD

**Issue by dhlee-jubilee** [melodyguan/enas#96]
The results of validation/test accuracy are able to be check through result of print, but how to check the pre-step values of accuracy(logits, or prediction value)? If the test set is imbalanced, other metrics should be applied not accuracy. I'd like to check the probability and calculate confusion matrix with val/test set.

(1) in "models.py", "micro_child.py", "general_child.py"
def _build_valid(self):
     logits = self._model(self.x_valid, False, reuse=True) 
     self.valid_logits = logits  ### added
     ... 
(2) in "models.py "
def eval_once(...)
if eval_set == "valid":
    val_logit = self.valid_logits  ### added
    ...
    valid_prob = sess.run(val_logit, feed_dict=feed_dict)

(did test as same as validation)

(3) in "main.py", fixed sequence of labels 
def bd(x, y, batch_size=4, isTrain=False):
    return tf.data.Dataset.from_tensor_slices((x, y)).shuffle(10).batch(batch_size).repeat()

def bd2(x, y, batch_size=FLAGS.batch_size, isTrain=False):
    return tf.data.Dataset.from_tensor_slices((x, y)).batch(batch_size).repeat()

train_dataset = bd(x_train, y_train, batch_size=FLAGS.batch_size, isTrain=True)
valid_dataset = bd2(x_valid, y_valid, batch_size=FLAGS.batch_size, isTrain=False)
test_dataset = bd2(x_test, y_test, batch_size=FLAGS.batch_size, isTrain=False)

However, the result of calculated accuracy from val_logits/test_logits don't same with the original accuracy of ENAS. 

I really cannot find the issue. Help me.
